citizen/quick_start_centos.md

```diff
@@ -0,0 +1,131 @@
+## Quick Start Guide for Citizen Node
+
+## Setting build environment
+ * install libraries
+
+ ```
+ $ sudo yum update
+ $ sudo yum install git wget zlib-devel bzip2 bzip2-devel readline-devel sqlite sqlite-devel openssl-devel \
+   xz xz-devel libffi-devel gcc gcc-c++ automake libtool lsof -y
+ ```
+
+## Python Installation
+
+Python 3.7.x
+
+ * Install **pyenv**
+ ```
+ $ curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash
+ ```
+
+ * Append the following commands to ~/.bash_profile
+ ```
+ export PATH="/home/centos/.pyenv/bin:$PATH"
+ eval "$(pyenv init -)"
+ eval "$(pyenv virtualenv-init -)"
+ ```
+ * Apply for the profile
+ ```
+ $ source ~/.bash_profile
+ ```
+ * Install **python 3.7.x**
+ ```
+ $ pyenv install 3.7.x
+ $ pyenv shell 3.7.x
+ $ python -V # check python version
+ ```
+
+## Make Virtual Environment
+ * Change working directory to citizen pack directory.
+ ```
+ $ cd citizen_pack
+ ```
+
+ * Install virtualenv
+ ```
+ $ pip3 install virtualenv
+ ```
+
+ * make virtual environment and apply
+ ```
+ $ virtualenv -p python venv
+ $ source venv/bin/activate
+ ```
+
+## RabbitMQ Installation
+ * Install erlang
+ ```
+ $ wget https://github.com/rabbitmq/erlang-rpm/releases/download/v21.3.7/erlang-21.3.7-1.el7.x86_64.rpm
+ $ sudo rpm -ivh erlang-21.3.7-1.el7.x86_64.rpm
+ ```
+
+ * Install RabbitMQ
+ ```
+ $ sudo rpm --import https://www.rabbitmq.com/rabbitmq-release-signing-key.asc
+ $ wget https://dl.bintray.com/rabbitmq/all/rabbitmq-server/3.7.7/rabbitmq-server-3.7.7-1.el7.noarch.rpm
+ $ sudo yum install rabbitmq-server-3.7.7-1.el7.noarch.rpm -y
+ ```
+
+ * Run RabbitMQ
+ ```
+ $ sudo systemctl enable rabbitmq-server
+ $ sudo systemctl start rabbitmq-server
+ ```
+
```

Comment:
I think it is better to provide RabbitMQ installation guide as what Loopchain `README.md` is doing:
https://github.com/icon-project/loopchain#requirements

Reply:
done